_includes/footer-sgb.html

```diff
@@ -4,29 +4,29 @@
 
         <div class="col-md-7 px-4 mt-3">
             <p class="text-white"><small>{{ site.description }}</small></p>
-            <p class="text-white"><small>{{
-                    site.data.translations['_includes']['footer.html']['last-updated'][site.lang] |
-                    default: 'Last updated' }} {{ site.time | date: '%Y' }}</small></p>
         </div>
 
-        <div class="col-md-5 mt-3 text-center">
-            {% if site.organization-logo-nav %} <p class="text-md-end">
+        <div class="col-md-12 mt-3 text-center">
+            <p class="text-white fw-bold" id="footer-credits">
+                {% if site.organization-logo-nav %}
                 <a href="{{ site.organization-link }}" target="_blank" rel="noopener">
                     <img id="footer-logo" class="img-fluid" src="{{ site.organization-logo-nav | relative_url }}"
                         alt="{{ site.organization-name | escape }} home">
                 </a>
-            </p>
             {% endif %}
-            <p class="text-white text-md-end" id="footer-credits">
-                <small><em>{{ site.data.translations['_includes']['footer.html']['built-with'][site.lang] | default:
+
+                <small><em><span aria-hidden="true">&#x2E31; </span>{{ site.data.translations['_includes']['footer.html']['built-with'][site.lang] | default:
```

Comment:
@mtwente the footer text is not small anymore, otherwise it seems great!

Reply:
@maehr Isn't it the same size as before?

Reply:
nope: https://beta.stadtgeschichtebasel.ch/

Reply:
Ah, adding `fw-bold` for "built-with" was on purpose, I thought it is a better fit with the width of the SGB logo glyphs.

Reply:
Ok, I liked it better before. Can you revert?

Reply:
I will green-light the PR because I am AFK. Please merge as soon as its fixed.

Thanks again for the PR. The platform looks amazing.